File: swagger/models/get_client_purchases_response.py
from .base_model import BaseModel

from .pagination_response import PaginationResponse
from .client_purchase_record import ClientPurchaseRecord

class GetClientPurchasesResponse(BaseModel):
	"""
	no description available

	Attributes:

		pagination_response
		Contains information about the pagination used.

		purchases
		Contains information that describes the item sold and the payment.
	"""

	input_map = {
		'PaginationResponse': ('pagination_response', PaginationResponse),
		'Purchases': ['purchases', ClientPurchaseRecord],
		}

	output_map = {
		'pagination_response': 'PaginationResponse',
		'purchases': 'Purchases',
		}

	pagination_response = None
	purchases = None

File: swagger/Models/GetClientPurchasesResponse.php
<?php

namespace App\Mindbody\Models;

/**
 * no description available
 */
class GetClientPurchasesResponse extends BaseModel
{
	/**
	 * Contains information about the pagination used.
	 * 
	 * @var PaginationResponse 
	 */
	public $paginationResponse;
	/**
	 * Contains information that describes the item sold and the payment.
	 * 
	 * @var ClientPurchaseRecord[] 
	 */
	public $purchases;

	protected function getInputMap()
	{
		return [
			'PaginationResponse' => ['paginationResponse', PaginationResponse::class, null],
			'Purchases' => ['purchases', 'array', ClientPurchaseRecord::class],
			];
	}

	protected function getOutputMap()
	{
		return [
			'paginationResponse' => ['PaginationResponse', PaginationResponse::class, null],
			'purchases' => ['Purchases', 'array', ClientPurchaseRecord::class],
			];
	}
}
